src/pipeline/mod.rs

```diff
@@ -17,7 +15,7 @@ impl Pipeline {
         Pipeline::new(pedals)
     }
 
-    fn new(pedals: Vec<pedal::Boxed>) -> Result<Self> {
+    pub fn new(pedals: Vec<pedal::Boxed>) -> Result<Self> {
```

Comment:
This is public now for use when making pipelines that function as individual `Pedal`s in a larger pipeline.